package.json

```diff
@@ -23,13 +23,15 @@
     "@blueprintjs/select": "^3.8.0",
     "@blueprintjs/table": "3.5.0",
     "@blueprintjs/timezone": "3.10.1",
+    "axios": "1.6.7",
```

Comment:
Can we use https://github.com/bitfinexcom/bfx-facs-http please?

Reply:
Will update to it a bit later, due to some compatibility issues that are currently found while using it in the UI

Reply:
Ok, makes sense.